pyproject.toml

```diff
@@ -27,6 +27,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.10"
+poetry = "^1.8.4"
```

Comment:
Unless you import from poetry at runtime it should not be specified as a dependency.

Reply:
Fix in #112